Hub updates

jwijay opened the pull request:
**Superficial updates to the following pages:**
* Sign up
* Reset password
* Project Files modals: Gdoc account not linked, unlink Github repository
* Project Sharing
* Account Create
* Account Profile
* Account Members
* Account Teams
* Team Members
* Team Projects

---

**Before: sign up (width)**
<img width="452" alt="Screenshot 2019-05-30 16 25 50" src="https://user-images.githubusercontent.com/10161095/58671745-92542880-82f8-11e9-8114-9268f34396cc.png">

**After: Sign up (width)**
<img width="540" alt="Screenshot 2019-05-30 16 30 39" src="https://user-images.githubusercontent.com/10161095/58671732-9122fb80-82f8-11e9-9cd6-f39f5eea9c23.png">

---

**Before: Reset password (spacing, title)**
<img width="566" alt="Screenshot 2019-05-30 16 25 40" src="https://user-images.githubusercontent.com/10161095/58671746-92542880-82f8-11e9-9c2e-18a59c9b689c.png">

**After: Reset password (spacing, title)**
<img width="931" alt="Screenshot 2019-05-30 16 30 32" src="https://user-images.githubusercontent.com/10161095/58671733-9122fb80-82f8-11e9-962a-96226ff51ad1.png">

---

**Before: Modal (button)**
<img width="693" alt="Screenshot 2019-05-30 16 13 28" src="https://user-images.githubusercontent.com/10161095/58671758-92ecbf00-82f8-11e9-9819-4e411d34a78b.png">

**After: Modal (button)**
<img width="723" alt="Screenshot 2019-05-30 16 16 00" src="https://user-images.githubusercontent.com/10161095/58671757-92ecbf00-82f8-11e9-893e-e83d9b14c1a0.png">

---
**Before: Unlink modal (copy wording/grammar)**
<img width="664" alt="Screenshot 2019-05-30 16 27 45" src="https://user-images.githubusercontent.com/10161095/58671743-91bb9200-82f8-11e9-881f-57837db1ef70.png">

**After: Unlink modal (copy wording/grammar)**
<img width="666" alt="Screenshot 2019-05-30 16 16 23" src="https://user-images.githubusercontent.com/10161095/58671756-92ecbf00-82f8-11e9-974a-829b88fce873.png">

---

**Before: Sharing (buttons, switch color)**
<img width="992" alt="Screenshot 2019-05-30 16 27 57" src="https://user-images.githubusercontent.com/10161095/58671742-91bb9200-82f8-11e9-9f7a-0963d7e603df.png">

**After: Sharing (buttons, switch color)**
<img width="1030" alt="Screenshot 2019-05-30 16 17 11" src="https://user-images.githubusercontent.com/10161095/58671755-92ecbf00-82f8-11e9-9ebf-ac6d869ba4e2.png">

---

**Before: Account Create (typo, spacing)**
<img width="1188" alt="Screenshot 2019-05-30 16 28 06" src="https://user-images.githubusercontent.com/10161095/58671740-91bb9200-82f8-11e9-8400-ae4cb64da512.png">

**After: Account Create (typo, spacing)**
<img width="1200" alt="Screenshot 2019-05-30 16 19 47" src="https://user-images.githubusercontent.com/10161095/58671754-92ecbf00-82f8-11e9-851c-fc6d8998e08d.png">

---

**Before: Account Profile (button, card)**
<img width="1192" alt="Screenshot 2019-05-30 16 28 16" src="https://user-images.githubusercontent.com/10161095/58671739-91bb9200-82f8-11e9-9d38-1ba0499696cd.png">

**After: Account Profile (button, card => box)**
<img width="1188" alt="Screenshot 2019-05-30 16 20 08" src="https://user-images.githubusercontent.com/10161095/58671753-92ecbf00-82f8-11e9-91a4-9440dbc178a4.png">

**After: Account Profile (box hover state fix)**
<img width="796" alt="Screenshot 2019-05-30 16 20 13" src="https://user-images.githubusercontent.com/10161095/58671752-92542880-82f8-11e9-8601-317a62bed50a.png">

---

**Before: Account Members (button, spacing)**
<img width="1196" alt="Screenshot 2019-05-30 16 28 23" src="https://user-images.githubusercontent.com/10161095/58671738-91bb9200-82f8-11e9-9947-8e4db268f479.png">

**After: Account Members (button, spacing)**
<img width="1191" alt="Screenshot 2019-05-30 16 20 51" src="https://user-images.githubusercontent.com/10161095/58671751-92542880-82f8-11e9-89c7-e6a37624875f.png">

---

**Before: Account Teams (cards, button)**
<img width="1186" alt="Screenshot 2019-05-30 16 28 29" src="https://user-images.githubusercontent.com/10161095/58671737-9122fb80-82f8-11e9-920a-b8e21b1a669c.png">

**After: Account Teams (cards => boxes, button)**
<img width="1199" alt="Screenshot 2019-05-30 16 21 12" src="https://user-images.githubusercontent.com/10161095/58671750-92542880-82f8-11e9-8f5b-f2f934acba6a.png">

---

**Before: Team Members (button, spacing)**
<img width="1199" alt="Screenshot 2019-05-30 16 29 15" src="https://user-images.githubusercontent.com/10161095/58671736-9122fb80-82f8-11e9-937d-e0437f0c35f1.png">

**After: Team Members (button, spacing)**
<img width="1197" alt="Screenshot 2019-05-30 16 21 46" src="https://user-images.githubusercontent.com/10161095/58671749-92542880-82f8-11e9-8823-d42e8e5f8991.png">

---

**Before: Team Projects (button, spacing)**
<img width="1188" alt="Screenshot 2019-05-30 16 29 23" src="https://user-images.githubusercontent.com/10161095/58671734-9122fb80-82f8-11e9-9dd2-00ee9cdb8f5f.png">

**After: Team Projects (button, spacing)**
<img width="1189" alt="Screenshot 2019-05-30 16 22 05" src="https://user-images.githubusercontent.com/10161095/58671748-92542880-82f8-11e9-853c-abe972f33301.png">

alex-ketch commented:
Really great round of cleanup @jwijay, looking much tighter overall! 👍

Some feedback:
- It seems we've been using a left-alignment for buttons for the most part, but the "Account Create" view is an outlier, with the "Create" button aligned to the right.
- For a future PR/design iteration, what do you think about testing a two-column or constrained width layout for places where we have a lot of help text (_Account Create_ and _Account Members_ from the screenshots above), to constrain the line-length a bit, and raise the actionable items in hierarchy/verticality a bit?

beneboy commented on director/accounts/templates/accounts/account_profile.html:
```diff
@@ -19,35 +19,28 @@
                 <div class="level-right">
                     <p class="level-item">
-                        <a class="button is-success" href="{% url 'project_create' %}?account_id={{ account.pk }}">
+                        <a class="button is-primary is-rounded is-outlined call-to-action" href="{% url 'project_create' %}?account_id={{ account.pk }}">
```
I think `call-to-action` is redundant now? What properties do we need for it – I didn't use it throughout any of my last round of changes.

beneboy commented on director/accounts/templates/accounts/account_teams.html:
```diff
@@ -15,30 +15,22 @@
             {% if account_permissions|account_permissions_contain:'administer' %}
                 <p class="level-item">
-                    <a class="button is-success" href="{% url 'account_team_create' account.id %}">
+                    <a class="button is-primary is-rounded is-outlined call-to-action" href="{% url 'account_team_create' account.id %}">
```
CTA again/

beneboy commented:
> * It seems we've been using a left-alignment for buttons for the most part, but the "Account Create" view is an outlier, with the "Create" button aligned to the right.

Anything that is generated from a Django from is probably doing this. They need to be made consistent.

jwijay commented:
Thanks for the feedback, guys!

I removed the CTA style from: Account List; Project List; Account Profile (New Project); Account Team (New Team); Sign out modal.

> * It seems we've been using a left-alignment for buttons for the most part, but the "Account Create" view is an outlier, with the "Create" button aligned to the right.

I updated the css so it's left-aligned by default. Pages impacted include:
* Project Settings: Meta-data
* Project FIles: Archive
* Project Files: Github Link
* Project Create
* Account Create
* Settings Change Username

> * For a future PR/design iteration, what do you think about testing a two-column or constrained width layout for places where we have a lot of help text (_Account Create_ and _Account Members_ from the screenshots above), to constrain the line-length a bit, and raise the actionable items in hierarchy/verticality a bit?

👍 That sounds good to me.
